src/app/shared/input-ref.directive.ts

```diff
@@ -0,0 +1,22 @@
+import { Directive, Input, HostBinding } from '@angular/core';
+import { AbstractControl } from '@angular/forms';
+
+@Directive({
+  selector: '[appInputRef]',
+})
+export class InputRefDirective {
+  @Input('appInputRef') control: AbstractControl;
```

Comment:
Nie wiem gdzie to wyczytales - ale pomysł PRO (mam nadzieje ze to nie jest kopiujwkleing tylko wiesz po co tu akurat dyrektywa jest) :)

Reply:
Dlugo sie z tym meczylem